service/controller/v18/encrypter/tls.go

```diff
@@ -1,151 +0,0 @@
-package encrypter
-
-import (
-	"bytes"
-	"compress/gzip"
-	"context"
-	"encoding/base64"
-
-	"github.com/giantswarm/legacycerts/legacy"
-	"github.com/giantswarm/microerror"
-)
```

Comment:
Again. Less indirection. Encrypter does only encryption. It doesn't know about certificates. This is done in `cloudconfig` package.